src/cli/Makefile

```
@@ -20,10 +21,15 @@ STACK       = -fstack-protector-all -Wstack-protector
 WARNS       = -Wall -Wextra -pedantic -Werror
 CFLAGS      = -fPIC $(OPTIMIZATION) $(STD) $(STACK) $(WARNS)
 DEBUG       = -g3 -DDEBUG=1
-LIBS        = -L /usr/lib -L $(LIB_DIR) -l $(LIB_NAME_EXT) -l $(LIB_NAME)
+
+ifeq ($(shell uname -s),Darwin)
+    LIBS = -L $(LIB_DIR) -l $(LIB_NAME_EXT) -l $(LIB_NAME)
+else
+    LIBS = -L $(LIB_DIR) -l:$(LIB_FILENAME) -l:$(LIB_FILENAME_EXT)
+endif
 
 ifeq ($(BUILD_TLS),yes)
-	CFLAGS += -DUSE_OPENSSL=1
+    CFLAGS+=-DUSE_OPENSSL=1
```

```suggestion
	CFLAGS+=-DUSE_OPENSSL=1
```
There is some tab issue here